src/Curation/Curation.model.ts

```diff
@@ -0,0 +1,40 @@
+import { Model, raw, SQL } from 'decentraland-server'
+import { CurationAttributes } from './Curation.types'
+
+export class Curation extends Model<CurationAttributes> {
+  static tableName = 'curations'
+
+  static getAll() {
```

Comment:
I've based this queries on others I found on the repo, yet I'm kind of worried about interpolating the values directly into the query, which might cause sql injection attacks, (in this case it's not possible because sanitization is done by the router). Do you know if this library supports some kind of sanitization. Eg: `query("WHERE col = $1", [value])`